src/main/java/com/puppycrawl/tools/checkstyle/checks/TrailingCommentCheck.java

```diff
@@ -201,14 +221,17 @@ public void beginTree(DetailAST rootAST) {
             final String line = getLines()[lineNo - 1];
             final String lineBefore;
             final TextBlock comment;
+            final int commentColNo;
             if (cppComments.containsKey(lineNo)) {
                 comment = cppComments.get(lineNo);
                 lineBefore = line.substring(0, comment.getStartColNo());
+                commentColNo = comment.getStartColNo();
             }
             else {
                 final List<TextBlock> commentList = cComments.get(lineNo);
                 comment = commentList.get(commentList.size() - 1);
                 lineBefore = line.substring(0, comment.getStartColNo());
+                commentColNo = comment.getStartColNo();
```

Comment:
`commentColNo = comment.getStartColNo();`
this does not depends on if condition, please inline its usage on line 245

Reply:
awesome feedback, fixed.